data/mainnet/tokens.js

```diff
@@ -7185,4 +7185,23 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['iFARM', 'DAI', 'WETH'],
   },
+  convex_cvxCRV: {
```

Comment:
Forgot to mention this before, but please add new vaults to the top of the file, below FARM and WETH in `tokens.js`. So in `pools.js` all the way on the top, in `tokens.js` just below FARM and WETH.